xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -52,12 +52,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The byte read-only span containing the compressed data.</param>
+        <param name="destination">When this method returns, a byte span containing the decompressed data.</param>
+        <param name="bytesConsumed">The total number of bytes that were read from <paramref name="source" />.</param>
+        <param name="bytesWritten">The total number of bytes that were written in the <paramref name="destination" />.</param>
+        <summary>Decompresses data that was compressed using the Brotli algorithm.</summary>
+        <returns>One of the enumeration values that can be returned from span-based operations that support processing of input contained in multiple discontiguous buffers.</returns>
```

Comment:
I think we should converge on the documentation for the "OperationStatus" based APIs that process a source into a destination span.

`that support processing of input contained in multiple discontiguous buffers.` <= this seems like unnecessary context.

Personally, I like the Bas64 API/parameter descriptions. Let's use that as a template (including for parameters like bytesWritten, bytesConsumed):
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.text.base64.decodefromutf8?view=netcore-3.0#System_Buffers_Text_Base64_DecodeFromUtf8_System_ReadOnlySpan_System_Byte__System_Span_System_Byte__System_Int32__System_Int32__System_Boolean_

Reply:
Does this sound good?:

```suggestion
        <returns>One of the enumeration values that indicates the status of the decoding operation.</returns>
```

Reply:
We aren't decoding here, we are decompressing. I would tailor the comments to be specific to this API rather than copying it as-is.

Reply:
```suggestion
        <returns>One of the enumeration values that indicates the status of the decompression operation.</returns>
```